src/main/java/teammates/ui/webapi/action/GetFeedbackSessionsAction.java

```diff
@@ -35,14 +45,22 @@ public ActionResult execute() {
 
         List<FeedbackSessionAttributes> feedbackSessionAttributes;
         if (courseId == null) {
-            boolean isInRecycleBin = getBooleanRequestParamValue(Const.ParamsNames.IS_IN_RECYCLE_BIN);
+            if (userInfo.isInstructor) {
```

Comment:
If a login entity is both a student and an instructor of the same course, he will never be able to get courses involved as a student.

Maybe we want to use the presence of the param `IS_IN_RECYCLE_BIN ` to distinguish the request.